packages/shared/lib/chrysalis.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why do we need this JSON?

Reply:
Its the JSON that will be used for the snapshot so it doesnt return a 404, but we can simply remove it and create it when we need it

Reply:
I would move it to a config folder or similar nearer the root of the repo so its more obvious.